CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/compose/composables/ContentCard.kt

```diff
@@ -0,0 +1,27 @@
+package co.nimblehq.coroutine.ui.screens.compose.composables
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Suppress("MagicNumber")
+@Composable
+fun ContentCard(
+    content: @Composable () -> Unit
+) {
+    Card(
+        shape = RoundedCornerShape(
+            topStart = 24.dp,
```

Comment:
I think we should extract all of the dimension value into one constant file. 🤔  Because I see it's used in many places and we add the `@Suppress("MagicNumber")`.

Reply:
Resolved 🙏🏼